src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java

```diff
@@ -213,6 +223,11 @@ public PaginatedResult getProjects(final Team team, final boolean excludeInactiv
                 .excludeInactive(excludeInactive)
                 .withTeam(team);
 
+        if (onlyRoot){
+            filterBuilder.excludeProjectsThatAreChildren();
+            pm.getFetchPlan().addGroup(Project.FetchGroup.ALL.name());
```

Comment:
The fetch plan should be set for `query`, otherwise it may have unintended impact on other queries performed by this `pm`.